pysat/utils/io.py

```diff
@@ -1580,6 +1580,9 @@ def inst_to_netcdf(inst, fname, base_instrument=None, epoch_name=None,
         included will be written to the file. If not listed
         and a value is NaN then that attribute simply won't be included in
         the netCDF4 file. (default=None)
+    export_pysat_info : bool
+        If True, platform, name, tag, and inst_id will be appended to the
+        metadata.  (default=True)
```

Comment:
```suggestion
    export_pysat_info : bool
        Appends the platform, name, tag, and inst_id to the metadata
        if True. Otherwise these attributes are lost. (default=True)
```